test/libsolidity/syntaxTests/array/length/too_large.sol

```diff
@@ -1,5 +1,8 @@
 contract C {
     uint[8**90] ids;
+    uint[2**256-1] okay;
+    uint[2**256] tooLarge;
```

Comment:
Can you add also `uint[uint(2**256-1)]`? Possibly as a separate test file with appropriate name and some more type conversion cases along with it.

I'm assuming this will still trigger an error due to the `!lengthValue` check.

Reply:
Done